packages/stark-ui/src/modules/table/components/dialogs/multisort.component.html

```diff
@@ -1,14 +1,15 @@
 <div class="header">
-	<h1 mat-dialog-title>Multi-Column Sorting </h1>
+	<h1 mat-dialog-title translate>STARK.MULTI_COLUMN_SORTING.TITLE</h1>
 	<button mat-mini-fab mat-icon-button color="primary" (click)="onAdd()" [disabled]="isAddDisabled">
-		<mat-icon svgIcon="playlist-plus" matTooltip="Add sorting rule"></mat-icon>
+		<mat-icon svgIcon="playlist-plus" [matTooltip]="'STARK.MULTI_COLUMN_SORTING.ADD_SORTING_LEVEL' | translate"></mat-icon>
 	</button>
 </div>
 <div mat-dialog-content>
 	<div class="rules">
-		<div *ngFor="let rule of rules; trackBy: trackRuleFn">
+		<div *ngFor="let rule of rules; trackBy: trackRuleFn; let index = index">
```

Comment:
You can use the "first" local variable directly instead of "index" to simplify a bit the code:
https://angular.io/api/common/NgForOf#local-variables.